spec/features/admin/affirm_payment_method_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe "Affirm payment method admin" do
+  stub_authorization!
+
+  before(:each) do
+    visit spree.admin_path
+    click_link "Settings"
+  end
+
+  context "admin creating a new payment method" do
```

Comment:
We introduced a new payment method, I want to make sure that I can select that in the admin interface.